Dockerfile

```diff
@@ -48,6 +48,10 @@ RUN apt-get -y install \
     latex-cjk-chinese-arphic-gbsn00lp latex-cjk-chinese-arphic-gkai00mp \
     fonts-symbola
 
+# Install imagemagick to support sphinx.ext.imgconverter
+# http://www.sphinx-doc.org/en/master/usage/extensions/imgconverter.html
+RUN apt-get -y install imagemagick
```

Comment:
Do we want a separate layer for just this dependency or should this be bundled up in another step? I think we need to reduce the number of layers here, and can squash a lot of our operations.

Reply:
I'm not sure how this affects the image in the end. I think it's related to the size of the resulting image but it will need further research from my side to have an opinion here.